From 1373a1ec0f3b0be9db080cdea6206a42659d477f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Fri, 19 Jul 2024 02:39:27 +0000
Subject: [PATCH] Update appdata.xml

- Fix screenshot and donation URLs
- Fix gettext domain
- Add URLs to translate and vcs-browser
---
 data/org.gnome.GnomeSubtitles.appdata.xml.in | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/data/org.gnome.GnomeSubtitles.appdata.xml.in b/data/org.gnome.GnomeSubtitles.appdata.xml.in
index 7380427..9d64315 100644
--- a/data/org.gnome.GnomeSubtitles.appdata.xml.in
+++ b/data/org.gnome.GnomeSubtitles.appdata.xml.in
@@ -24,10 +24,12 @@
   </categories>
   <url type="homepage">https://gnomesubtitles.org</url>
   <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnome-subtitles/issues/</url>
-  <url type="donation">https://www.gnomesubtitles.org/donate</url>
+  <url type="donation">http://sourceforge.net/p/gnome-subtitles/donate</url>
+  <url type="translate">https://l10n.gnome.org/module/gnome-subtitles/</url>
+  <url type="vcs-browser">https://gitlab.gnome.org/GNOME/gnome-subtitles</url>
   <screenshots>
     <screenshot type="default">
-      <image>https://gnomesubtitles.org/files/screenshots/Screenshot-GnomeSubtitles-VideoTimesTranslation.png</image>
+      <image>https://a.fsdn.com/con/app/proj/gnome-subtitles/screenshots/GnomeSubtitles.png</image>
       <caption>Main window</caption>
     </screenshot>
   </screenshots>
@@ -40,7 +42,7 @@
     <mimetype>text/x-subviewer</mimetype>
   </mimetypes>
   <project_group>GNOME</project_group>
-  <translation type="gettext">gnome-subtitles</translation>
+  <translation type="gettext">org.gnome.GnomeSubtitles</translation>
   <content_rating type="oars-1.1" />
   <releases>
     <release version="1.8" date="2022-06-24" />
-- 
GitLab

